src/esp/assets/GenericInstanceMeshData.h

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <Magnum/GL/Buffer.h>
+#include <Magnum/GL/Mesh.h>
+
+#include <Magnum/GL/Texture.h>
+#include "BaseMesh.h"
+#include "esp/core/esp.h"
+
+namespace esp {
+namespace assets {
+
+Magnum::GL::Texture2D createInstanceTexture(float* data, const int texSize);
```

Comment:
I would add a brief comment here as the purpose of this function is not immediately obvious

Reply:
Copied the comment on the implementation to here